.eslintrc.json

```diff
@@ -14,8 +14,9 @@
           "MemberExpression": 1
       }],
       "object-curly-spacing": ["error", "always"],
-      "react/jsx-uses-react": "error",   
-      "react/jsx-uses-vars": "error" 
+      "react/jsx-uses-react": "error",
+      "react/jsx-uses-vars": "error" ,
+      "eol-last": 2
```

Comment:
@j-zhao FYI: this already exists in the google config we're extending: https://github.com/google/eslint-config-google/blob/master/index.js#L200

https://github.com/music-markdown/music-markdown/pull/27/files#diff-df39304d828831c44a2b9f38cd45289cL2